tests/sqlfluff_templater/linter_test.py

```diff
@@ -6,11 +6,15 @@
 import pytest
 
 from sqlfluff.core import Linter, FluffConfig
-from tests.sqlfluff_templater.fixtures.dbt.templater import DBT_FLUFF_CONFIG, project_dir  # noqa: F401
+from tests.sqlfluff_templater.fixtures.dbt.templater import (
+    DBT_FLUFF_CONFIG,
+    project_dir,
+)  # noqa: F401
 
 
-@pytest.mark.parametrize(
-    "path", ["models/my_new_project/disabled_model.sql", "macros/echo.sql"]
+@pytest.mark.parametrize("path", ["models/my_new_project/disabled_model.sql", "macros/echo.sql"])
+@pytest.mark.skip(
```

Comment:
Feel free to just remove this test. I initially copied all the tests from `sqlfluff-templater-dbt`, then removed several that weren't relevant.

BTW, it's nice that the new templater handles some of this better than `sqlfluff-templater-dbt`. It would be cool to apply some of this back to _that_ templater.